modules/sdk/project-templates/project-templates-form-field/src/main/resources/archetype-resources/src/main/resources/content/Language.properties

```diff
@@ -1 +1,4 @@
+#if (${liferayVersion.startsWith("7.1")})
+${artifactId}-description=${artifactId} Description
```

Comment:
For the other portlets, we have adopted the pattern `${className} Description`  Do we want to do the same here?  Let me know what you think.